_episodes/00-intro.md

```diff
@@ -77,6 +77,7 @@ The programs to be introduced are:
 1. [_R_]({{ page.root }}/01-r-plotting): a statistical analysis and data management program, 
 1. [_RStudio_]({{ page.root }}/01-r-plotting): a graphical interface to use R, and
 1. [_R Markdown_]({{ page.root }}/05-r-markdown): a method for writing reproducible reports. 
+
 We'll use these tools to manage data, perform basic statistical analyses, and make pretty plots!
 
 We will not try to make you an expert or even proficient with any of them, but we
```

Comment:
Idk why but I don't love this sentence. Maybe make it something like "By the end of this workshop, we hope you will ..." or "This work shop won't make you an expert but we hope to provide you with..."